controllers/quadpid/calibration/calib_quadpid.c

```diff
@@ -314,6 +320,16 @@ static void *ctrl_quadpid_speed_thread_cmd_linear_speed(void *arg)
     return 0;
 }
 
+/**
+ * @brief Angular speed characterization command
+ *
+ * Run command in a dedicated thread
+ *
+ * @param[in]   argc    Number of arguments
```

Comment:
Usually, parameter descriptions start with a lower case letter.
At least, it must be consistent everywhere.